### DISCUSSION TITLE
[BUG] Album display order wrong in App

### DISCUSSION THREAD

**Discussion by Nordlicht-13** [immich-app/immich#8734] [Feature Request]
### The bug

The album display order in App is always from newest to oldest, even if you choose in the Immich.web oldest to newest.

### The OS that Immich Server is running on

Immich kupernetic app on TrueNAS

### Version of Immich Server

V1. 101.0

### Version of Immich Mobile App

V1. 101.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Standard docker-compose.yml
```

### Your .env content

```Shell
Standard
```

### Reproduction steps

```bash
1. Change the album display order in immich.web to oldest first
2. View same album in immich.app
3. In immich.app it shiws up with newest first
...
```

**Comment by mmomjian** [comment]
Thank you for bringing this up. Converting this to a discussion as I do not believe we currently do not have the feature of sorting  albums on the app.

**Reply by Nordlicht-13** [reply]
I thought it's the same album in immich.web and immich.app.

**Reply by mmomjian** [reply]
The album is the same, but the option to change the view order is not available in the mobile app as far as I am aware. Therefore the option chosen on the web does not carry over to the mobile app.

Do you mean the order of the albums themselves, or the order of the pictures in the albums? To change the order of the albums, you can click on the "Created date" button and change it to whatever you want. Clicking on the same option will reverse the order.

**Reply by Nordlicht-13** [reply]
I was talking about the order of the pictures in the albums.

**Reply by mmomjian** [reply]
Makes sense. Currently this is not supported in the mobile app.

**Reply by scottrbaxter** [reply]
Is there any plan to offer better sorting options? i'm honestly surprised that this software is so robust but something as basic as photo sorting has next to no support. the one option that is provided is buried in a menu and not available across devices... seems like a very strange design choice.

what's most surprising to me is that in recent months we've gotten a very robust option to sort albums themselves, with an easy and intuitive pulldown menu at the top of the page... on both web and mobile app. open any of these albums, and users are forced to look at all photos in reverse timeline. hidden in a menu is an option to change the display order, which needs to be done individually for each and every album, and doesn't transfer over to mobile... in fact, there's not even an option to toggle this at all on mobile, it's simply missing from this menu.

do people really prefer to look at wedding photos, sporting events, concerts, etc backwards? i just don't understand the logic. the only place it seems remotely useful to see "newest first" would be in a cameraroll... which is purely out of convenience because users have an ever growing collection of photos in the list. this also has nothing to do with albums.

terribly sorry that this sounds like a rant... i really do wish that i could better contribute to this project and offer some guidance. is there anything else that i (we?) can do to bring this feature request to the roadmap so that it can get implemented sooner than later? tbh, this is the biggest limitation for me to replace any/every other photo management solution and go with Immich full time... but i can't even think of another solution on the market without the ability to sort photos by name/size/type/date/uploaded time/ascending/descending. if album sort is already implemented, why not utilize that feature for pictures within albums, too?

**Reply by alextran1502** [reply]
@scottrbaxter there are a lot to do, so eventually we will get to this

**Reply by scottrbaxter** [reply]
@alextran1502 I appreciate the follow-up and am happy to hear it's on the roadmap already! i've asked about this same feature in a few previous issues and never got a response. the app is ridiculously good as is... very few minor things that could use some polish, but all in all you and your team have done incredible work. i'll keep on the lookout for future releases!
